src/mobu/flock.py

```diff
@@ -81,9 +82,14 @@ async def start(self) -> None:
             await monkey.start(self._scheduler)
 
     async def stop(self) -> None:
-        """Stop all the monkeys."""
-        for monkey in self._monkeys.values():
-            await monkey.stop()
+        """Stop all the monkeys.
```

Comment:
This isn't going to work.  At least, that was my experience trying this with gather.

At the very least you are going to want to keep track of which monkeys correctly shut down the first time, and iterate on that quickly-shrinking list.  You will be catching a number of exceptions if you fly a few dozen or more monkeys at once.

Reply:
If this doesn't work, there's a bug in stop (and indeed there are several problems with how stop works right now, such as the fact that it's entirely disconnected from the running event loop and races with it). I don't want to hide those bugs; I want to fix them properly.